packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -35,13 +40,63 @@ export const TranslationSidebar = ({
     ['BDB', 'LSJ'].includes(resource)
   );
   const lexiconEntry = lexiconResource?.entry ?? '';
-  const { t } = useTranslation(['common', 'translate']);
+
+  const translatorNotesQuery = useQuery(
+    ['verse-translator-notes', language, verse.id],
+    () => apiClient.verses.findTranslatorNotes(verse.id, language)
+  );
+  const translatorNote = translatorNotesQuery.isSuccess
+    ? translatorNotesQuery.data.data[word.id]
+    : null;
 
   const tabTitles = ['translate:lexicon', 'translate:notes'];
   if (showComments) {
     tabTitles.push('translate:comments');
   }
 
+  const userCan = useAccessControl();
+
+  const canEditNote = userCan('translate', {
+    type: 'Language',
+    id: language,
+  });
+
+  // Only update the content shown in the text editor when the word ID changes.
+  const [originalNoteContent, setOriginalNoteContent] = useState('');
+  const [noteContent, setNoteContent] = useState('');
+  const wordId = useRef('');
```

Comment:
I tried that, it breaks with the debounce logic whenever `setNoteContent` is called, because `noteContent` will get set to something that trims off the spaces, like I mentioned before.

Reply:
If we could get rid of the debounce logic, it would really simplify the code.

Reply:
I push up #324 with an attempt at what I was talking about. This stuff is tricky to get right